devops/jobs/SSLExpirationCheck.groovy

```diff
@@ -52,9 +52,9 @@ class SSLExpirationCheck{
                             'Git repo containing edX configuration.')
                     stringParam('CONFIGURATION_BRANCH', extraVars.get('CONFIGURATION_BRANCH', 'master'),
                             'e.g. tagname or origin/branchname')
-                    stringParam('SYSADMIN_REPO', extraVars.get('SYSADMIN_REPO'),
-                            'Git repo containing sysadmin configuration which contains the ssl expiration check script.')
-                    stringParam('SYSADMIN_BRANCH', extraVars.get('SYSADMIN_BRANCH', 'master'),
+                    stringParam('MONITORING_SCRIPTS_REPO', extraVars.get('MONITORING_SCRIPTS_REPO', 'https://github.com/edx/monitoring-scripts'),
```

Comment:
The second part of this tuple is just in case the variable isn't defined right?

Reply:
My instinct is to not have it be `extraVars.get('MONITORING_SCRIPTS_REPO')`, but I don't feel that srongly.